### PR TITLE
AC Stub Test - LRAC-8328 | master

### PR DESCRIPTION
**Pull request by rafaellajordao**
https://issues.liferay.com/browse/LRAC-8328

### CODE REVIEW

**Comment by ViniLopes20** [comment]
Hi, @rafaellajordao. Commits always starting with LRAC-XXXX (XXX -> Your branch number).

because it won't give an error when calling `ci:forward`

**Comment by alexcandidos** [comment]
hi @rafaellajordao , please see my comment here https://github.com/alexcandidos/liferay-portal/pull/510#issuecomment-893875039

**Comment by rafaellajordao** [comment]
Fixed! @alexcandidos and @ViniLopes20!

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesCohortAnalysis.testcase [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		ACUtils.tearDownAC();
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8327 | Test Summary: Assert Sites Cohort Analysis sort Anonymous Visitors"
+	@ignore = "true"
+	@priority = "5"
+	test AssertSitesCohortAnalysisSortAnonymousVisitors {
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesCohortAnalysis.testcase:30]: hi @rafaellajordao , this tc is a priority 3 on the test map.

**Review by alexcandidos — CHANGES_REQUESTED** [review]
Hi @rafaellajordao , please update one of the tcs priorities. Thanks!

**Comment by alexcandidos** [comment]
Hi @rafaellajordao , your last two PRs were with the wrong ticket id, they were not pointing to the ticket you had crated to commit the stubs. 

this was pointing to LRAC-8325 instead of LRAC-8328

**Review by alexcandidos — APPROVED** [review]
Hi @rafaellajordao , Thanks for working on it!

The code looks good to me!

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**